source/Nevermore/Nevermore.csproj

```diff
@@ -45,7 +45,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="System.Diagnostics.Contracts" Version="4.3.0" />
     <PackageReference Include="System.ValueTuple" Version="4.4.0" />
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="2.1.4" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="4.1.0" />
```

Comment:
I'd recommend against using the sqlClient retry provider for now until we have a much more stable build environment. Currently, I'm only aware of this option for opening connections (not beginning transactions, but this advice applies to both in case that comes to light). Perhaps we don't want to make that change at all regardless.